odoo/tools/translate.py

```diff
@@ -445,7 +460,7 @@ def __call__(self, source):
                 if cr:
                     # Try to use ir.translation to benefit from global cache if possible
                     env = odoo.api.Environment(cr, odoo.SUPERUSER_ID, {})
-                    res = env['ir.translation']._get_source(None, ('code','sql_constraint'), lang, source)
+                    res = env['ir.translation']._get_source(None, ('code'), lang, source)
```

Comment:
This is not a tuple!
```suggestion
                    res = env['ir.translation']._get_source(None, ('code',), lang, source)
```

Reply:
Good catch, actually a string is also accepted as parameter